src/sparsezoo/utils/registry.py

```diff
@@ -64,10 +84,16 @@ class ImageNetDataset(Dataset):
     class Cifar(Dataset):
         pass
 
+    Note: the name will be standardized for lookup in the registry.
+    For example, if a class is registered as "cifar_dataset" or
+    "cifar dataset", it will be stored as "cifar-dataset". The user
+    will be able to load the class with any of the three name variants.
+
     # register with multiple aliases
-    @Dataset.register(name=["cifar-10-dataset", "cifar-100-dataset"])
+    @Dataset.register(alias=["cifar-10-dataset", "cifar_100_dataset"])
```

Comment:
I think I'm confused here, where does the actual registry name get set? Is "datasetname-dataset" a default if a name isn't provided?

Reply:
in this case the name would be `Cifar`